cf-java-logging-support-core/beats/request-metrics/docs/fields.asciidoc

```diff
@@ -294,13 +298,13 @@ never the case.
 
 type: date
 
-example: 2015-01-24 14:06:05.071000
+example: 2015-01-24 14:06:05.071000+00:00
 
 format: YYYY-MM-DDTHH:MM:SS.milliZ
 
```

Comment:
The given format and the example do not match up.
If this is intentional, maybe clarify why they do not match up.
same for following two timestamps

Reply:
This format is introduced by the python script generating this documentation. :-(